spec/features/user_edits_a_spot_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+feature "User edits a spot" do
+
+# As a user I want to be able to edit a spot I created
+# so that the information is up-to-date.
+# Acceptance criteria:
+# User can edit the spot they created.
+
+  scenario 'I want to edit a study spot' do
+    spot1 = Spot.find_or_create_by(
```

Comment:
Don't use `find_or_create_by` for any of these tests, just use `create` or `create!`